app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= stylesheet_link_tag "recipes", "data-turbo-track": "reload" %>
+
+<h1>Recipes</h1>
+
+<% @recipes.each do |recipe| %>
+  <div class="recipe" data-id="<%= recipe.id %>">
+    <div class="left">
+      <h3><%= recipe.name %></h3>
+      <div class="remove-btn">
+        <% if recipe.user == current_user %>
+          <%= form_with(model: recipe, url: recipe_path(recipe), method: :delete) do |form| %>
+            <%= form.hidden_field :_method, value: "delete" %>
+            <%= form.submit 'Remove', class: 'button-white-black' %>
+          <% end %>
+        <% end %>
+      </div>
+    </div>
+    
+    <%= link_to recipe_path(recipe), class: "right" do %>    
+      <p><%= recipe.description %></p>
+    <% end %>
+  </div>
+<% end %>
+
+<%= link_to "New Recipe", new_recipe_path, class: "add-btn" %>
```

Comment:
Hi team, you have worked really hard you this application. Thank you for your dedication in delivering a project that meets all the project requirements 🚀 

### Suggestions
[OPTIONAL] There are a few things that can be added to the application to improve privacy and user experience.

**Actions:**

When a user is not logged-in, it is best to hide action buttons. This will ensure that users are not able to perform certain actions like creating a `New recipe` and `modify` the ingredients on a recipe. 
Although the buttons do not permit a user to perform an action, it is not advisable leave buttons that throw errors in an application. Rather create a condition that request the user to sign in to continue with the action.

![image](https://github.com/danifromecuador/recipes/assets/38088818/f397852a-1a1b-47a1-8819-29cff082ce6c)

![image](https://github.com/danifromecuador/recipes/assets/38088818/043e3a12-a9be-4028-bff5-cd47f8e0e50a)

To enhance user experience, the `logout` button can also be made invisible when a user is not logged in. Or otherwise you can create a condition that displays `login` when a user in not logged-in and `logout` when a user is logged in.